docs/api/resource.rst

```diff
@@ -192,9 +194,9 @@ HTTP Status Codes
 -----------------
 
 * ``200 OK``: The request was processed
-* ``304 Not Modified``: Collection did not change since value in ``If-Modified-Since`` header
+* ``304 Not Modified``: Collection did not change since value in ``If-None-Match`` header
 * ``400 Bad Request``: The request querystring is invalid
-* ``412 Precondition Failed``: Collection changed since value in ``If-Unmodified-Since`` header
```

Comment:
Off the top of my head, a client could send a PUT with `If-None-Match: *` to make sure that only creates a new document if none exists, but never overwrites an existing one, would also fail with 412 and not 304. So I think 304s are for read actions and 412s are for write actions. Would have to look it up to be sure, though.